tests/backup/backup_locked_bucket_test.go

```diff
@@ -406,3 +406,233 @@ var _ = Describe("{LockedBucketResizeOnRestoredVolume}", func() {
+				})
+				Step("Create schedule backup after initializing volume resize", func() {
+					log.InfoD("Create schedule backup after initializing volume resize")
+					ctx, err := backup.GetAdminCtxFromSecret()
+					dash.VerifyFatal(err, nil, "Fetching px-central-admin ctx")
+					scheduleName = fmt.Sprintf("%s-schedule-%v", BackupNamePrefix, time.Now().Unix())
+					scheduleNames = append(scheduleNames, scheduleName)
+					labelSelectors := make(map[string]string)
+					err = CreateScheduleBackup(scheduleName, SourceClusterName, backupLocationName, backupLocationUID, []string{namespace},
+						labelSelectors, orgID, "", "", "", "", periodicSchedulePolicyName, periodicSchedulePolicyUid, ctx)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of schedule backup with schedule name [%s]", scheduleName))
+					firstScheduleBackupName, err := GetFirstScheduleBackupName(ctx, scheduleName, orgID)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the name of the first schedule backup [%s]", firstScheduleBackupName))
+				})
+				Step("Verifying backup success after initializing volume resize", func() {
+					log.InfoD("Verifying backup success after initializing volume resize")
+					ctx, err := backup.GetAdminCtxFromSecret()
+					dash.VerifyFatal(err, nil, "Fetching px-central-admin ctx")
+					firstScheduleBackupName, err := GetFirstScheduleBackupName(ctx, scheduleName, orgID)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the name of the first schedule backup [%s]", firstScheduleBackupName))
+					_, err = backupSuccessCheck(firstScheduleBackupName, orgID, 0, 0, ctx)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying the success of recent backup named [%s]", firstScheduleBackupName))
+					allScheduleBackupNames, err := Inst().Backup.GetAllScheduleBackupNames(ctx, scheduleName, orgID)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching all schedule backups %v", allScheduleBackupNames))
+					log.InfoD("Waiting for 15 minutes for the next schedule backup to be triggered")
+					time.Sleep(15 * time.Minute)
+					recentBackupName, err := GetLatestScheduleBackupName(ctx, scheduleName, orgID)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching recent backup %v", recentBackupName))
+					_, err = backupSuccessCheck(recentBackupName, orgID, 0, 0, ctx)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying the success of recent backup named [%s]", recentBackupName))
+				})
+			}
+		}
+	})
+	JustAfterEach(func() {
+		defer EndPxBackupTorpedoTest(contexts)
+		ctx, err := backup.GetAdminCtxFromSecret()
+		dash.VerifySafely(err, nil, "Fetching px-central-admin ctx")
+		for i := 0; i < len(scheduleNames); i++ {
+			scheduleUid, err := GetScheduleUID(scheduleNames[i], orgID, ctx)
+			dash.VerifySafely(err, nil, fmt.Sprintf("Fetching uid of schedule named [%s]", scheduleNames[i]))
+			err = DeleteSchedule(scheduleNames[i], scheduleUid, periodicSchedulePolicyNames[i], periodicSchedulePolicyUids[i], orgID)
```

Comment:
In one of the future PRs DeleteSchedule will not delete the schedule policy automatically.

Reply:
Thanks!